### PR TITLE
Assets/multilingual font compression

### PR DESCRIPTION
**Pull request by tkoleary**
## Description
-This PR fixes bugs in the loading of the following fonts that were already in assets but not displaying properly:
--Hebrew
--Myanmar
--Greek
--Cyrillic
-Re-factors Latin by adding an extra font to cover Latin characters used by other languages like Vietnamese
-Adds the following fonts supported by Mass.gov that were not in assets yet:
--Armenian
--Devanagari
--Gujarati
--Khmer
--Lao
--Malayalam
--Tamil
--Telugu
--Thai
-Updates the readme in fonts with the python scripts for subsetting fonts
-Updates the typography page in core 

## Related Issue / Ticket
- [JIRA issue - DP-20436]
- [JIRA issue - DP-20682]
- [JIRA issue - DP-20686]

## Steps to Test
1. Pull the branch and run rush start:core
2. Open firefox and go to http://localhost:6006/?path=/docs/foundation-typography--page
3. look at the  screenshots which provide some insight on how you can highlight the specific characters loading in a font using the FF inspectors fonts panel
4. Open the inspector and the fonts tab and check that all of the fonts are loading

### From Core
Screen Shot 2020-12-23 at 4 06 58 PM
Screen Shot 2020-12-23 at 4 07 17 PM
Screen Shot 2020-12-23 at 4 07 32 PM
Screen Shot 2020-12-23 at 4 07 43 PM
Screen Shot 2020-12-23 at 4 09 42 PM
Screen Shot 2020-12-23 at 4 09 51 PM
Screen Shot 2020-12-23 at 4 11 07 PM

#### @TODO
-Gujarati has no VF
-All of the newly added fonts still need eot fallbacks
-There are no py scripts in the readme yet for Gujarati, Devanagari, Tamil, Telugu, and Thai
-CJK fonts  (Chinese Japanese, Korean) are still not done
-Mono has no eot fallbacks yet
-The latin-ext VF does not have non-variable fallbacks or eot and the woff2 has a bug so it's commented out, but the woff works.

### CODE REVIEW

**Comment by tkoleary** [comment]
Down to 7 fails,1 in Patternlab and 6 in React.
Cant check backstop because it fails locally for me. Docker issue I suspect.

File: packages/core/stories/tokens/typography/Typography.stories.mdx [hunk 1]
```diff
@@ -90,130 +83,131 @@ Using these fonts for your print and digital assets is a subtle but compelling w
...
 ---
 
+#### Left-to-right
+
 #### Armenian script
 
-<!-- @todo investigate why Note font is not rendereing here -->
 <p className="lang-label">Armenian: </p>
-<p lang="hy" className="font-specimen">Բարի գալուստ Մասաչուսեթս</p>
+<p lang="hy" className="font-specimen">Ու նայիրեան աղջիկների հեզաճկուն պա՛րն եմ սիրում։</p>
```
Comment (clairesunstudio) [inline packages/core/stories/tokens/typography/Typography.stories.mdx]: what is this text? can we revert it to the original text -- which is the title of the mass.gov homepage?

File: packages/core/stories/tokens/typography/Typography.stories.mdx [hunk 2]
```diff
@@ -90,130 +83,131 @@ Using these fonts for your print and digital assets is a subtle but compelling w
...
-<!-- @todo investigate why Note font is not rendereing here -->
 <p className="lang-label">Chinese: </p>
-<p lang="zh-Hans" className="font-specimen">欢迎来到马萨诸塞州</p>
-
-#### Chinese, traditional script
-
-<!-- @todo investigate why Note font is not rendereing here -->
-<p className="lang-label">Chinese: </p>
-<p lang="yue-Hant" className="font-specimen">歡迎來到馬薩諸塞州</p>
+<p lang="zh-Hans" className="font-specimen">微風迎客，軟語伴茶</p>
```
Comment (clairesunstudio) [inline packages/core/stories/tokens/typography/Typography.stories.mdx]: see comment above, same for all the other languages

File: packages/core/stories/tokens/typography/Typography.stories.mdx [hunk 3]
```diff
@@ -102,13 +102,13 @@ If your project needs to support an addtional language, please [submit a feature
 #### Armenian script
 
 <p className="lang-label">Armenian: </p>
-<p lang="hy" className="font-specimen">Ու նայիրեան աղջիկների հեզաճկուն պա՛րն եմ սիրում։</p>
+<p lang="hy" className="font-specimen">Բարի գալուստ Մասաչուսեթս</p>
```
Comment (clairesunstudio) [inline packages/core/stories/tokens/typography/Typography.stories.mdx:105]: My original comment was about reverting all the font specimen text changes in this file, not just for these two languages. Can you revert all of them instead?

**Comment by clairesunstudio** [comment]
UAT link --> https://mayflower.digital.mass.gov/core/b/assets/multilingual-font-compression/index.html?path=/docs/foundation-typography--page

File: packages/core/stories/tokens/typography/Typography.stories.mdx [hunk 4]
```diff
@@ -90,130 +83,131 @@ Using these fonts for your print and digital assets is a subtle but compelling w
   }
 </div>
 
-## Languages by Script
+## Language support
```
Comment (clairesunstudio) [inline packages/core/stories/tokens/typography/Typography.stories.mdx]:
```suggestion
## Language Support
```

File: packages/core/stories/tokens/typography/Typography.stories.mdx [hunk 5]
```diff
@@ -90,130 +83,131 @@ Using these fonts for your print and digital assets is a subtle but compelling w
...
-The only fonts that are rendering with Noto are: Latin, Greek, Cyrillic, Khmer, Arabic and  Hebrew.
--->
+Arabic, Armenian, Chinese (Simplified), Filipino, French, German, Greek, Gujarati, Haitian Creole, Hebrew, Hindi, Italian, Japanese, Khmer, Korean, Lao, Malayalam, Persian, Polish, Portuguese, Russian, Spanish, Tamil, Telugu, Thai, Ukrainian, Urdu, and Vietnamese.
 
-### Left-to-right
+The foolowing do not yet have webfont support: Chinese (Simplified), Japanese, and Korean.
+
+If your project needs to support an addtional language, please [submit a feature request](https://github.com/massgov/mayflower/issues/new/choose) on Github.
+
+### Languages by script
```
Comment (clairesunstudio) [inline packages/core/stories/tokens/typography/Typography.stories.mdx]:
```suggestion
### Languages by Script
```

File: packages/core/stories/tokens/typography/Typography.stories.mdx [hunk 6]
```diff
@@ -223,27 +217,27 @@ The only fonts that are rendering with Noto are: Latin, Greek, Cyrillic, Khmer,
 #### Thai script
 
 <p className="lang-label">Thai: </p>
-<p lang="Thai" className="font-specimen">ยินดีต้อนรับสู่แมสซาชูเซตส์</p>
+<p lang="Thai" className="font-specimen">เป็นมนุษย์สุดประเสริฐเลิศคุณค่า กว่าบรรดาฝูงสัตว์เดรัจฉาน จงฝ่าฟันพัฒนาวิชาการ อย่าล้างผลาญฤๅเข่นฆ่าบีฑาใคร ไม่ถือโทษโกรธแช่งซัดฮึดฮัดด่า หัดอภัยเหมือนกีฬาอัชฌาสัย ปฏิบัติประพฤติกฎกำหนดใจ พูดจาให้จ๊ะ ๆ จ๋า ๆ น่าฟังเอยฯ</p>
 
-### Right-to-Left
+#### Right-to-Left
```
Comment (clairesunstudio) [inline packages/core/stories/tokens/typography/Typography.stories.mdx]:
```suggestion
### Right-to-Left
```

File: packages/core/stories/tokens/typography/Typography.stories.mdx [hunk 7]
```diff
@@ -90,130 +83,131 @@ Using these fonts for your print and digital assets is a subtle but compelling w
...
-### Left-to-right
+The foolowing do not yet have webfont support: Chinese (Simplified), Japanese, and Korean.
+
+If your project needs to support an addtional language, please [submit a feature request](https://github.com/massgov/mayflower/issues/new/choose) on Github.
+
+### Languages by script
 
 ---
 
+#### Left-to-right
```
Comment (clairesunstudio) [inline packages/core/stories/tokens/typography/Typography.stories.mdx]:
```suggestion
### Left-to-right
```

File: packages/core/stories/tokens/typography/Typography.stories.mdx (hunk 3 above)
Comment (clairesunstudio) [reply packages/core/stories/tokens/typography/Typography.stories.mdx:105]: The specimen text should match what's in production --> https://mayflower.digital.mass.gov/core/index.html?path=/docs/foundation-typography--page

File: packages/core/stories/tokens/typography/Typography.stories.mdx [hunk 8]
```diff
@@ -196,20 +187,25 @@ The only fonts that are rendering with Noto are: Latin, Greek, Cyrillic, Khmer,
 <p lang="pl" className="font-specimen">Witamy w Massachusetts</p>
 
 <p className="lang-label">Portuguese: </p>
-<p lang="pt" className="font-specimen">Bem vindo a Massachusetts</p>
+<p lang="es" className="font-specimen">Bienvenido a massachusetts</p>
```
Comment (clairesunstudio) [inline packages/core/stories/tokens/typography/Typography.stories.mdx]:
```suggestion
<p lang="pt" className="font-specimen">Bem vindo a Massachusetts</p>
```

File: packages/assets/static/fonts/readme.md [hunk 9]
```diff
@@ -11,50 +11,504 @@ To use:
 4. If not, copy the entire block for a script, paste it to the command line and enter
 5. Run git status to test if the font files were updated
 
-# Latin fonts
+#### Troubleshooting
 
-<!--@todo: update with eot s-->
+- The most common error is that the font is not found which means the first line of the command doesnt match a file name in the directory you are in.
```
Comment (clairesunstudio) [inline packages/assets/static/fonts/readme.md]:
```suggestion
- The most common error is that the font is not found which means the first line of the command doesn't match a file name in the directory you are in.
```

File: packages/assets/static/fonts/readme.md [hunk 10]
```diff
@@ -11,50 +11,504 @@ To use:
...
+- If the command does not workas one block, try running them one at a time (make sure to exlude the &&).
```
Comment (clairesunstudio) [inline packages/assets/static/fonts/readme.md]:
```suggestion
- If the command does not work as one block, try running them one at a time (make sure to exclude the `&&`).
```

File: packages/assets/static/fonts/readme.md [hunk 11]
```diff
@@ -11,50 +11,504 @@ To use:
...
+- Don't forget to cd into the right direecctory for the script first.
```
Comment (clairesunstudio) [inline packages/assets/static/fonts/readme.md]:
```suggestion
- Don't forget to cd into the right directory for the script first.
```

File: packages/core/stories/tokens/typography/Typography.stories.mdx (hunk 8 above)
Comment (clairesunstudio) [reply packages/core/stories/tokens/typography/Typography.stories.mdx]: fix portuguese

File: packages/core/stories/tokens/typography/Typography.stories.mdx [hunk 12]
```diff
@@ -238,7 +234,7 @@ The only fonts that are rendering with Noto are: Latin, Greek, Cyrillic, Khmer,
 <p lang="fa" className="font-specimen">به ماساچوست خوش آمدید</p>
 
 <p className="lang-label rtl">Urdu: </p>
-<p lang="ur" className="font-specimen">میساچوسٹس میں آپ کا استقبال ہےد</p>
+<p lang="fa" className="font-specimen">به ماساچوست خوش آمدید</p>
```
Comment (clairesunstudio) [inline packages/core/stories/tokens/typography/Typography.stories.mdx]:
```suggestion
<p lang="ur" className="font-specimen">میساچوسٹس میں آپ کا استقبال ہےد</p>
```

File: packages/core/stories/tokens/typography/Typography.stories.mdx [hunk 13]
```diff
@@ -90,22 +83,24 @@ Using these fonts for your print and digital assets is a subtle but compelling w
...
+The following languages are officially supported by [Mass.gov](https://www.mass.gov):
 
-<!--
-@todo look into font rendering issues below.
-The only fonts that are rendering with Noto are: Latin, Greek, Cyrillic, Khmer, Arabic and  Hebrew.
--->
+Arabic, Armenian, Chinese (Simplified), Filipino, French, German, Greek, Gujarati, Haitian Creole, Hebrew, Hindi, Italian, Japanese, Khmer, Korean, Lao, Malayalam, Persian, Polish, Portuguese, Russian, Spanish, Tamil, Telugu, Thai, Ukrainian, Urdu, and Vietnamese.
 
-### Left-to-right
+The foolowing do not yet have webfont support: Chinese (Simplified), Japanese, and Korean.
```
Comment (clairesunstudio) [inline packages/core/stories/tokens/typography/Typography.stories.mdx]:
```suggestion
The following do not yet have Noto Sans webfont support: Chinese (Simplified), Japanese, and Korean.
```

**Review by clairesunstudio — APPROVED** [review]

**Comment by clairesunstudio** [comment]
patternlab backstop false positive --> https://app.circleci.com/pipelines/github/massgov/mayflower/2962/workflows/bddc9c0e-63e0-4ecc-9a4f-b174594683d5/jobs/41746/artifacts

File: packages/assets/scss/00-base/_fonts-langs-vars.scss [hunk 14]
```diff
@@ -31,8 +31,9 @@ $japanese:				U+A5, U+4E00-9FFF, U+30??, U+FF00-FF9F; // Kanji, Hiragana, Kataka
 $kannada:					U+0C80-0CFF;
 $kayah:						U+A900-A92F;
 $khmer:						U+1780-17FF, U+19E0-19FF; // Khmer and symbols
-$lao:							U+0E80-0EFF;
-$latin:						U+????;
+$lao:							U+????; // U+0E80-0EFF;
```
Comment (tom-fleming) [inline packages/assets/scss/00-base/_fonts-langs-vars.scss:34]: @clairesunstudio Just wanted to check whether this is intentional or was instead intended as a placeholder that didn't get follow up

**Pull request merged**